samples/Basic_WiFi/app/application.cpp

```diff
@@ -28,17 +28,17 @@ void listNetworks(bool succeeded, BssList list)
 }
 
 // Will be called when WiFi station was connected to AP
-void connectOk()
+void connectOk(IPAddress ip, IPAddress mask, IPAddress gateway)
 {
 	debugf("I'm CONNECTED");
-	Serial.println(WifiStation.getIP().toString());
+	Serial.println(ip.toString().c_str());
```

Comment:
You are right. The standard string should work just fine `size_t Print::println(const String &s)` . Thanks for pointing this out.